src/js/components/Form/stories/ValidateOnBlur.js

```diff
@@ -23,6 +24,18 @@ const Example = () => (
                 if (name && name.length === 1) return 'must be >1 character';
                 return undefined;
               },
+              name => {
+                if (name === 'good')
```

Comment:
what will be a good way to reveal this easter egg?

Reply:
word of mouth :)